Resources/Prototypes/_Moffstation/PersonalItems/Wearables/FrostWinters/Frost_Winters/frostmedicalwebbing.yml

```
@@ -14,9 +14,8 @@
 
 - type: loadout
   id: PersonalItemFrostMedicalWebbing
-  storage:
-    belt:
-    - PersonalItemFrostMedicalWebbing
+  inhand:
+  - PersonalItemFrostMedicalWebbing
```

Sorry, it took me a bit, but this is nominally the "correct" way to do this
```suggestion
  equipment:
    belt: PersonalItemFrostMedicalWebbing
```
but because we can't override loadout contents (specifically the normal medbelt here), it means one of the belts ends up on the floor.

---

[I knew I'd done this before](https://github.com/moff-station/moff-station-14/pull/38#discussion_r2030228284)...

> I'll be fixing it
If only 😭